src/main/java/seedu/address/model/person/Name.java

```diff
@@ -10,7 +10,8 @@
 public class Name {
 
     public static final String MESSAGE_CONSTRAINTS =
-            "Names should only contain alphanumeric characters and spaces, and it should not be blank";
+            "Names should only contain alphanumeric characters and spaces where the first"
+                    + "letter is capitalised or a number, and it should not be blank";
```

Comment:
It would be good if remove the number part as alphanumeric means alphabets and numbers. 
It would be good to include a space so that the `first` and the `letter` don't merge together.

Reply:
Got it!